templates/vote-status.md

```diff
@@ -1,6 +1,6 @@
 ## Vote status
 
-So far `{{ "{:.2}"|format(results.in_favor_percentage) }}%` of the users with binding vote are in favor (passing threshold: `{{ results.pass_threshold }}%`).
+So far `{{ "{:.2}"|format(results.in_favor_percentage) }}%` of the users with binding vote are in favor and `{{ "{:.2}"|format(results.against_percentage) }}%` are against (passing threshold: `{{ results.pass_threshold }}%`).
```

Comment:
Could you also update the `vote-closed` template? That is the one we render when the vote is closed. It's slightly different than the one used to display the vote status while the vote is still open (the one you updated).

https://github.com/cncf/gitvote/blob/4636393256cf6e72f34b87a94bcf4e16aa3ece5d/templates/vote-closed.md#L7